class/defaults.yml

```diff
@@ -1,3 +1,34 @@
 parameters:
   lieutenant:
-    namespace: syn-lieutenant
+    namespace: lieutenant
+    operator:
+      version: v0.5.3
+      manifest_url: https://raw.githubusercontent.com/projectsyn/lieutenant-operator/${lieutenant:operator:version}/deploy
+      image: 'docker.io/projectsyn/lieutenant-operator:${lieutenant:operator:version}'
+      common_labels:
+        'app.kubernetes.io/name': 'lieutenant-operator'
+        'app.kubernetes.io/part-of': 'project-syn'
+      default_deletion_policy: Archive
+      deletion_protection: false
+      default_global_git_repo: https://github.com/projectsyn/getting-started-commodore-defaults
+      vault:
+        enabled: true
+        addr: vault.todo
+        path: kv
+    api:
+      version: v0.6.0
+      image: 'docker.io/projectsyn/lieutenant-api:${lieutenant:api:version}'
+      manifest_url: https://raw.githubusercontent.com/projectsyn/lieutenant-api/${lieutenant:api:version}/deploy
+      common_labels:
+        'app.kubernetes.io/name': 'lieutenant-api'
+        'app.kubernetes.io/part-of': 'project-syn'
+      default_githost: ''
+      steward_image: docker.io/projectsyn/steward:v0.3.1
+      ingress:
+        host: lieutenant.todo
+        annotations: {}
+      users:
+        - kind: ServiceAccount
+          name: api-access-vshn-portal-prod
+    tenant_rbac: []
+    githosts: []
```

Comment:
Having `tenants` and `githosts` as a lists makes it impossible to edit existing entries in the hierarchy. 

You may want to consider using the following format instead (example for `githosts`):

```
githosts:
  vshn-gitlab:
    endpoint: ...
    token: ...
    host_keys: ...
```

and work with the resulting object as follows

```
[
  kube.Secret(name) { 
    spec: {
      stringData: {
        endpoint: githosts[name].endpoint,
        ...
      }
    }
  }
  for name in std.objectFields(githosts)
]

Reply:
I didn't realize that there is no support for removing entries from list.